docs/stapi-client/index.md

```diff
@@ -0,0 +1,14 @@
+# stapi-client
+
+A Python client (CLI tool) for interfacing with a [Satellite Tasking API (STAPI)](https://github.com/stapi-spec) server.
+
+!!! note
+
+    This is a work in progress.
+
+## Installation
+
+```shell
+uv pip install pystapi-client
```

Comment:
Not sure if we want to force **uv** down onto users:

```suggestion
python -m pip install pystapi-client
```

Reply:
😱 🤔 👌🏽